stack/stack.py

```diff
@@ -1,4 +1,5 @@
 class Stack(object):
+
```

Comment:
@singhpratyush sir these were the issues raised by coala, I asked above if I should fix them.